sample-queries/sample-queries.json

```diff
@@ -1426,6 +1426,23 @@
         "tip": "This query requires your user id and another user's id.  To find your user id & another user's id, you can run: 1) GET https://graph.microsoft.com/v1.0/me/ 2) GET https://graph.microsoft.com/v1.0/users",
         "skipTest": false
     },
+    {
+        "id": "8cd7a323-478e-4828-878f-7e6962728a9b",
+        "category": "Microsoft Teams",
+        "method": "POST",
+        "humanName": "send activity notification",
+        "requestUrl": "/v1.0/teams/{id}/sendActivityNotification",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-sendactivitynotification?view=graph-rest-1.0&tabs=http",
+        "headers": [
+            {
+                "name": "Content-Type",
+                "value": "application/json"
+            }
+        ],
+        "postBody": "{\r\n    \"topic\": {\r\n    \"source\": \"entityUrl\",\r\n    \"value\": \"https://graph.microsoft.com/v1.0/teams/b126aba2-a1fb-4e03-acc3-410b5d5af2dc\"\r\n    },\r\n    \"activityType\": \"pendingFinanceApprovalRequests\",\r\n    \"previewText\": {\r\n    \"content\": \"Internal spending team has a pending finance approval requests\"\r\n    },\r\n    \"recipient\": {\r\n    \"@odata.type\": \"microsoft.graph.aadUserNotificationRecipient\",\r\n    \"userId\": \"c8c95113-2dea-457d-9da4-e963d3f7a98b\"\r\n    },\r\n    \"templateParameters\": [\r\n    {\r\n    \"name\": \"pendingRequestCount\",\r\n    \"value\": \"5\"\r\n    }\r\n    ]\r\n    }",
```

Comment:
Nice job! For the post body, I don't think values such as b126aba2-a1fb-4e03-acc3-410b5d5af2dc and c8c95113-2dea-457d-9da4-e963d3f7a98b should be in there. The sample query in the documentation uses place holders like {teamId} so that may be better. While the documentation may not use a place holder for userid, I think it might be a good idea as well